_posts/bad-at-to-do-lists-now-what.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+header_image: /img/posts/bad-at-to-do-lists-now-what/header.jpg
+title: "Lifehacking 101: I'm bad at to-do lists, now what?"
+author: Mauro Servienti
+synopsis: "When it comes to to-do lists, I'm terrible. How do I get better at managing my activities and time? Context switching seems to be the real culprit. I'm pretty sure I won't get any better at to-do lists."
+tags:
+- Time management
+- To-Do lists
+- Context switching
+---
+
+I already wrote about how bad I am at to-do lists in ["I'm a procrastinator, I fail at to-do lists."](https://milestone.topics.it/2019/01/23/i-m-a-procrastinator-i-fail-at-to-do-lists.html).
+
+When it comes to to-do lists, I'm still terrible. I want to dedicate a few words to what I'm doing in an attempt to get better at managing my time; I'm not sure I'll get better at managing to-do lists, though.
+
+I've been running on zero-notifications for the last five or more years. Zero-notifications means that my computer, or any other device, have notifications turned off for everything except the following:
+
+- The calendar on the computer and phone
+- Phone calls and messages, including text messages, Signal, and WhatsApp. All groups, without exceptions, get muted on day one.
+Nothing else. And when it comes to phone calls and messages, my iPhone turns on "Do Not Disturb" automatically from 6 PM to  8 AM, in which case only 6 or 7 numbers can dial in. That is it.
+
+Running on zero-notifications is a relief; with no pressure coming from badges or any other sort of distracting and pressure-designed crap, my day is already doing great.
+
+However, as already said in the above-linked article, the day in Particular Software is a bit particular (no pun intended). Time zones, multiple task forces, wearing more than one hat ranging from "product management" to support, all participate in the risk of making the day a bit of a mess. 
+
+## Please welcome context switching 
+
+One of the side effects of wearing multiple hats throughout the day is the rise of context switching. Let's do some extreme math to put the issue in context; if we were to have only 5-minute long tasks and an 8-hour workday, the hypothetical result is 96 completed tasks: 12 tasks every hour, times 8 hours. Now, factor in multiple hats, and again for the sake of the sample, let's imagine that each assignment requires a different hat and that to change it (context switch), it takes our brain about 5 minutes. In such a scenario, we'll be able to complete 50% of the 96 tasks a day—a colossal cutoff.
+
+One could argue a zero-context switching approach would be an ideal world. Probably. I guess it depends on the person and the task. There are things that I enjoy a lot, like writing code or writing in general, that bores me very quickly; after a couple of hours in the best case, my mind starts digressing, signaling it's time for a change.
```

Comment:
> There are things that I enjoy a lot, like writing code or writing in general, that bores me very quickly

This looks contradictory. Do you mean you enjoy writing code but that writing in general bores you?

Reply:
I mean enjoy both of them, but both of them bore me very quickly. So, I wanna write code but not for 8 hours a day

Reply:
In that case:

> There are things that I enjoy a lot but that bore me very quickly, like writing code or writing in general.